src/ui_fsmenu/main.cc

```diff
@@ -19,184 +19,564 @@
+		if (newest_singleplayer) {
+			filename_for_continue_ = newest_singleplayer->filename;
+			singleplayer_.add(
+			   _("Continue Playing"), FullscreenMenuBase::MenuTarget::kContinueLastsave, nullptr,
+			   false,
+			   (boost::format("%s<br>%s<br>%s<br>%s<br>%s<br>%s<br>") %
+			    g_style_manager->font_style(UI::FontStyle::kTooltipHeader)
+			       .as_font_tag(
+			          /* strip leading "save/" and trailing ".wgf" */
+			          filename_for_continue_.substr(
+			             kSaveDir.length() + 1, filename_for_continue_.length() - kSaveDir.length() -
+			                                       kSavegameExtension.length() - 1)) %
+			    (boost::format(_("Map: %s")) % g_style_manager->font_style(UI::FontStyle::kTooltip)
+			                                      .as_font_tag(newest_singleplayer->mapname))
+			       .str() %
+			    (boost::format(_("Win Condition: %s")) %
+			     g_style_manager->font_style(UI::FontStyle::kTooltip)
+			        .as_font_tag(newest_singleplayer->wincondition))
+			       .str() %
+			    (boost::format(_("Players: %s")) % g_style_manager->font_style(UI::FontStyle::kTooltip)
+			                                          .as_font_tag(newest_singleplayer->nrplayers))
+			       .str() %
+			    (boost::format(_("Gametime: %s")) %
+			     g_style_manager->font_style(UI::FontStyle::kTooltip)
+			        .as_font_tag(newest_singleplayer->gametime))
+			       .str() %
+			    /** TRANSLATORS: Information about when a game was saved, e.g. 'Saved: Today, 10:30' */
+			    (boost::format(_("Saved: %s")) % g_style_manager->font_style(UI::FontStyle::kTooltip)
+			                                        .as_font_tag(newest_singleplayer->savedatestring))
+			       .str())
+			      .str(),
+			   "C");
+		}
+	}
+
+	multiplayer_.add(_("Online Game"), FullscreenMenuBase::MenuTarget::kMetaserver, nullptr, false,
+	                 _("Join the Widelands lobby"), "J");
+	multiplayer_.add(_("Online Game Settings"), FullscreenMenuBase::MenuTarget::kOnlineGameSettings,
+	                 nullptr, false, _("Log in as a registered user"), "U");
+	multiplayer_.add(_("LAN / Direct IP"), FullscreenMenuBase::MenuTarget::kLan, nullptr, false,
+	                 _("Play a private online game"), "P");
+
+	singleplayer_.set_label(_("Single Player…"));
+	multiplayer_.set_label(_("Multiplayer…"));
+	singleplayer_.set_tooltip(
+	   as_tooltip_text_with_hotkey(_("Begin or load a single-player campaign or free game"), "S"));
+	multiplayer_.set_tooltip(
+	   as_tooltip_text_with_hotkey(_("Play with your friends over the internet"), "M"));
+
+	replay_.set_title(_("Watch Replay"));
+	replay_.set_tooltip(as_tooltip_text_with_hotkey(_("Watch the replay of an old game"), "R"));
+
+	editor_.set_title(_("Editor"));
+	editor_.set_tooltip(as_tooltip_text_with_hotkey(_("Launch the map editor"), "E"));
+	addons_.set_title(_("Add-Ons"));
+	addons_.set_tooltip(
+	   as_tooltip_text_with_hotkey(_("This feature is still under development"), "A"));
```

Comment:
```suggestion
	   as_tooltip_text_with_hotkey("This feature is still under development", "A"));
```

We can always localize this if addons should not get finished for the next release.